spec/job/live_system_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+describe 'a live system', :if => (! ENV['LIVE_CLUSTER_CONFIG'].nil?) do
```

Comment:
```diff
-describe 'a live system', :if => (! ENV['LIVE_CLUSTER_CONFIG'].nil?) do
+describe 'a live system', :if => ENV['LIVE_CLUSTER_CONFIG'] do
or
+describe 'a live system', :unless => ENV['LIVE_CLUSTER_CONFIG'].nil? do
```

Reply:
You should also add `:order => :defined` so that you guarantee these tests run in order.

https://relishapp.com/rspec/rspec-core/v/3-8/docs/configuration/overriding-global-ordering#running-a-specific-example-group-in-order

Reply:
Updated.